doc/source/commands/runprogram.rst

```diff
@@ -0,0 +1,226 @@
+.. _running:
+
+Running Programs
+================
+
+.. contents:: Contents
+    :local:
+    :depth: 2
+
+KerboScript supports running executing code saved in files on the local or
+archive disks. The terms "program" and "script" are generally used
+interchangably when describing this behavior. Programs can either be run using
+one of the :ref:`runfunctions` or the :ref:`run`.  Supported file types are:
```

Comment:
```
:ref:`run`
```
will always be replaced with the heading title (I liked that choice so that it would automatically change if I revised the title).  But you can always modify the display label like this 
```
:ref:`my label here<run>`
```
That system will actually work for any of the `:role:` declarations, including structures and functions.
```
:func:`max(1,2)<max>`
:key:`RUN ONCE<RUN>`
```